packages/core/src/App/Components/Elements/LiveChat/live-chat.jsx

```diff
@@ -0,0 +1,170 @@
+                            landing_company_shortcode,
+                            currency,
+                            residence,
+                            first_name,
+                            last_name,
+                        } = JSON.parse(client_information) || {};
+                        session_variables = {
+                            ...(loginid && { loginid }),
+                            ...(landing_company_shortcode && { landing_company_shortcode }),
+                            ...(currency && { currency }),
+                            ...(residence && { residence }),
+                            ...(email && { email }),
+                        };
+                        // prefill name and email fields
+                        window.LiveChatWidget.call('set_session_variables', session_variables);
+                        window.LiveChatWidget.call('set_customer_email', email);
+                        window.LiveChatWidget.call('set_customer_name', `${first_name} ${last_name}`);
+                        // prefill name and email fields after chat has ended
+                        window.LC_API.on_chat_ended = () => {
+                            window.LiveChatWidget.call('set_customer_email', email);
+                            window.LiveChatWidget.call('set_customer_name', `${first_name} ${last_name}`);
+                        };
+                    }
+                } else {
+                    // client not logged in
+                    // clear name and email fields
+                    window.LiveChatWidget.call('set_customer_email', ' ');
+                    window.LiveChatWidget.call('set_customer_name', ' ');
+                    // clear name and email fields after chat has ended
+                    window.LC_API.on_chat_ended = () => {
+                        window.LiveChatWidget.call('set_customer_email', ' ');
+                        window.LiveChatWidget.call('set_customer_name', ' ');
+                    };
+                }
+            });
+        }
+    };
+
+    const livechatDeletion = () =>
+        new Promise(resolve => {
+            if (window.LiveChatWidget) {
+                window.LiveChatWidget.on('ready', () => {
+                    try {
+                        if (window.LiveChatWidget.get('customer_data').status !== 'chatting') {
+                            window.LiveChatWidget.call('destroy');
+                            resolve();
+                        }
+                    } catch (e) {
+                        resolve();
+                    }
+                });
+            } else {
+                resolve();
+            }
+        });
+
+    const handleHistoryChange = () => {
+        livechatDeletion().then(() => {
+            liveChatInitialization().then(() => {
+                setReload(true);
+                setLiveChatInteractive(true);
+                setReload(false);
+            });
+        });
+    };
+
+    return (
+        <>
```

Comment:
can we use `React.Fragment` instead?

Reply:
Yup sure 👌...btw <> is the short syntax for Fragment. Do you mean to say we're not allowed to use short syntax?

Reply:
@suthesan-deriv , it's not that it's not allowed, it's mainly for consistency. `<>` is just the synthetic sugar for `<React.Fragment>`, previous versions of babel (when we initially setup the project) didn't transpile this properly so we decided to go with `React.Fragment` instead to make things safe.
Plus, if u use `<>`, I remember in VSCode, it wont let you code fold the block. (I think this worked in TSX but not in JSX)

Reply:
I see..noted on that @zaki-binary 👌...Thank you 🙏